timemachine/potentials/nonbonded.py

```diff
@@ -19,11 +19,24 @@
 )
 
 
-def switch_fn(dij, cutoff):
-    return jnp.power(jnp.cos((jnp.pi * jnp.power(dij, 8)) / (2 * cutoff)), 2)
+def switch_fn(dij, cutoff=1.2):
+    """heuristic switching function
 
+    intended to:
+    * have {f, f', f''} go to 0 at cutoff
+    * keep "switch_fn(dij) * erfc(beta * dij)" as close as possible to "erfc(beta * dij)"
+        for the range dij in [0, 1.2), for beta = 2.0
 
-from typing import Optional
+    usage notes:
+    * express "no cutoff" as cutoff=np.inf
```

Comment:
This conflicts with the TODO, maybe remove this line?

Reply:
Good catch -- removed in https://github.com/proteneer/timemachine/pull/1290/commits/2a5e045e983fc457617428dbded5677fb4c9f00e